### DISCUSSION TITLE
PHP version 8.2?

### DISCUSSION THREAD

**Discussion by maxwellhibbert** [duncanmcclean/simple-commerce#869] [Help]
Is PHP 8.2 a suggestion or a requirement?
I have a project I was planning to use the plugin for, but the client's hosting setup currently offers PHP up to 8.1 only.

**Comment by duncanmcclean** [comment]
It's a requirement.

**Comment by duncanmcclean** [comment]
As of Simple Commerce [v5.2.0](https://github.com/duncanmcclean/simple-commerce/releases/tag/v5.2.0), PHP 8.1 is now supported again.
